src/modules/keyboardmanager/ui/EditKeyboardWindow.cpp

```diff
@@ -40,15 +40,22 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
         isEditKeyboardWindowRegistrationCompleted = true;
     }
 
+    // Find center screen coordinates
+    RECT desktopRect;
+    GetClientRect(GetDesktopWindow(), &desktopRect);
+    // Calculate resolution dependent window size
+    int windowWidth = KeyboardManagerConstants::DefaultEditKeyboardWindowWidth * desktopRect.right;
+    int windowHeight = KeyboardManagerConstants::DefaultEditKeyboardWindowHeight * desktopRect.bottom;
+
```

Comment:
We should probably use a more DPI aware way to calculate screen real state instead of just resolution.

Reply:
I couldn't find another good alternate on C++. The docs say that the values passed to CreateWindow are "device-independent" which made me think that it would scale with DPI and resolution, but it turns out it didn't. This was the closest solution I could find. Let me know if you have any other suggestions, if not we can make that change later.